utils/default-helpers/src/test/scala/pl/touk/nussknacker/engine/util/functions/CollectionUtilsSpec.scala

```diff
@@ -189,9 +189,9 @@ class CollectionUtilsSpec extends AnyFunSuite with BaseSpelSpec with Matchers {
     evaluateType("#COLLECTION.sum({#int, #long})", variables) shouldBe "Number".valid
   }
 
-  test("sum should fall back to Double on unknown Number type") {
```

Comment:
Please add tests:
1. doing the same but using simple plus operator
2. showing that we behave different than before in the runtime

Reply:
I've added tests

Reply:
What about tests mentioned in 2. point?